examples/getting-started-movielens/03-Training-with-HugeCTR.ipynb

```diff
@@ -3,7 +3,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !mv *.model {MODEL_DIR}
</pre><p>this is not os independent. you'll need <code>glob</code> and <code>shutil</code>    <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>